platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -87,6 +87,8 @@
                 dialog.getModalBody().css('padding', '0');
 
                 dialog.open();
+
+                dialog.getModal().prev().css('z-index', dialog.getModal().css('z-index'));
```

Comment:
@sebbrudzinski e.g. in task import we call first modal using html arguments (like here: http://www.w3schools.com/bootstrap/bootstrap_ref_js_modal.asp) and then second using LoadingModal. Both are using BootstrapDialog, but LoadingModal is checking that only one instance is opened.